src/components/SidebarFilter/_sidebar-filter.scss

```diff
@@ -33,6 +43,7 @@
 .cc-sidebar-filter__tags-list-item {
   background-color: var(--colour-blue-05);
   color: var(--colour-blue-60);
+  font-size: calc(12/16 * var(--text-base-size)); // we don't have this size in _typography.scss
```

Comment:
You could always specify this as a CSS variable either scoped globally in `:root` (in this file still) or locally in this block

Locally scoped variables is something I'm still getting my head around but here's some further info:
https://www.freecodecamp.org/news/everything-you-need-to-know-about-css-variables-c74d922ea855/#scoping-css-variables